src/generic/datavgen.cpp

```diff
@@ -910,6 +910,33 @@ class wxDataViewMainWindow: public wxWindow
     // assumes that all columns were modified, otherwise just this one.
     bool DoItemChanged(const wxDataViewItem& item, int view_column);
 
+    // Return whether the item has more than one column that have values
+    bool IsItemMultivalued(const wxDataViewItem& item) const
+    {
+        unsigned int numColsWithValue = 0;
+        unsigned int cols = GetOwner()->GetColumnCount();
+        const wxDataViewModel* model = GetModel();
+        for ( unsigned int i = 0; i < cols; i++ )
+        {
+            if ( model->HasValue(item, i) )
+                numColsWithValue++;
+            if ( numColsWithValue > 1 )
+                return true;
+        }
+        return false;
+    }
+
+    // find first column with value
+    wxDataViewColumn * FindFirstColumnWithValue(const wxDataViewItem& item) const
+    {
+        unsigned int cols = GetOwner()->GetColumnCount();
+        const wxDataViewModel* model = GetModel();
+        for ( unsigned int i ; i < cols; i++ )
```

Comment:
```suggestion
        for ( unsigned int i = 0; i < cols; i++ )
```
Unless I'm missing something, there is a bug here due to `i` not being initialized at all.

Reply:
Hmm, this got resolved, but the variable remained uninitialized. I'm adding ` = 0` here.